### PR TITLE
Webview2 

### PR DESCRIPTION
**Pull request by Killerherts**
This pull request add webview2 and brings client up to date with most recent features in uwp for xbox.

In testing this has increased the max quality from 15mbps as well as adding support for x265 direct playing. 

This is still mouse navigation. I would like to add programmatic focus in the future, but would rather not hold back the whole thing over it.

### CODE REVIEW

File: Jellyfin/Package.appxmanifest [hunk 1]
```diff
@@ -30,15 +30,15 @@
...
       <uap:VisualElements
-        DisplayName="Jellyfin"
+        DisplayName="jellfin-webView2"
         Square150x150Logo="Assets\Square150x150Logo.png"
         Square44x44Logo="Assets\Square44x44Logo.png"
         Description="Jellyfin"
         BackgroundColor="transparent">
         <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png"/>
-        <uap:SplashScreen Image="Assets\SplashScreen.png" BackgroundColor="#101010" />
+        <uap:SplashScreen Image="Assets\SplashScreen.png" />
```
Comment (crobibero) [inline Jellyfin/Package.appxmanifest:41]: Why remove the background color? Now it's a bright blue

File: Jellyfin/Package.appxmanifest [hunk 2]
```diff
@@ -30,15 +30,15 @@
   <Applications>
     <Application Id="App"
       Executable="$targetnametoken$.exe"
-      EntryPoint="Jellyfin.App">
+      EntryPoint="jellyfin.App">
       <uap:VisualElements
-        DisplayName="Jellyfin"
+        DisplayName="jellfin-webView2"
```
[the remark this answers is not in the document]
Comment (Killerherts) [reply Jellyfin/Package.appxmanifest]: done

**Review by crobibero — APPROVED** [review]
Changes look fine to me, leaving it for @anthonylavado to finish reviewing on an xbox :)

**Comment by dfederm** [comment]
@Killerherts curious, are you planning on revamping the UWP app? I recently started using Jellyfin and noticed on my Xbox that I couldn't watch in 4K so was considering poking around to see if that was something I could fix. Seems like you've already started this work though :).

**Comment by Killerherts** [comment]
> @Killerherts curious, are you planning on revamping the UWP app? I recently started using Jellyfin and noticed on my Xbox that I couldn't watch in 4K so was considering poking around to see if that was something I could fix. Seems like you've already started this work though :).

This is what this pull request was for. I do not believe I had passed Microsofts approval process and made it out 2 consoles though. 

@anthonylavado Am I correct in this assumption or do I need to make changes for approval for Ms store.

**Comment by anthonylavado** [comment]
I have to check the latest report and get back to you.

**Comment by Killerherts** [comment]
> I have to check the latest report and get back to you.

Did we get a response from MS?

**Comment by Killerherts** [comment]
@anthonylavado sorry to bother but can I get an update on this

**Pull request merged**